shared/src/vm/wasm/run.rs

```diff
@@ -690,6 +721,7 @@ mod tests {
         let mut gas_meter = VpGasMeter::new_from_tx_meter(
             &TxGasMeter::new_from_sub_limit(TX_GAS_LIMIT.into()),
         );
+        let _invalid_sig = false;
```

Comment:
are these unused declarations needed?

Reply:
Ah no they are typos from the very first version of the sentinel system, I'll remove them 👍🏻